scripts/create-change-file.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+ 
+let fs = require('fs');
+let inquirer = require('inquirer');
+let path = require('path')
+let guid = require('guid');
+
+let packages = [];  // keeps track of package.json files that have been found
+let answers = {};
+let data = [];
+
+// Gets all the package.json files under the current executing directory
+let getPackages = (dir, done) => {
+    fs.readdir(dir, (err, files) => {
+        let remaining = files.length;
+
+        if (remaining === 0) {
+            done();
+        }
+
+        files.forEach((file) => {
```

Comment:
The word `file` doesn't really make sense here because it's a file or a directory, right?